pom.xml

```diff
@@ -292,6 +292,16 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
arrow-vector-8.0.0-SNAPSHOT defines arrow-memory-netty and arrow-memory-unsafe with test scope, so they are not imported in the snowflake project. Adding them as direct dependencies so we can use them.